bedrock/mozorg/templates/mozorg/about/policy/transparency/jan-jun-2022.html

```diff
@@ -0,0 +1,470 @@
+          <td>0</td>
+          <td>7</td>
+        </tr>
+        <tr>
+          <th scope="row">Pocket</th>
+          <td>0</td>
+          <td>0</td>
+          <td>0</td>
+        </tr>
+        <tr>
+          <th scope="row">Other Services</th>
+          <td>0</td>
+          <td>0</td>
+          <td>0</td>
+      </tbody>
+    </table>
+
+  </div>
+</section>
+
+<section id="personal-data" class="c-collapsible-section">
+  <h2 class="c-collapsible-section-heading" data-accordion-role="tab">Personal Data Requests</h2>
+
+  <div data-accordion-role="tabpanel">
+    <p>In the Reporting Period, we received 10,716 requests.</p>
+
+    <table class="mzp-u-data-table">
+      <thread>
+        <th scope="col">Service</th>
+        <th scope="col">Received</th>
+      </thread>
+      <tbody>
+        <tr>
+          <th scope="row">Mozilla</th>
+          <td>7,842</td>
+        </tr>
+        <tr>
+          <th scope="row">Pocket</th>
+          <td>2,874</td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</section>
+
+<section id="targeted-advertising" class="c-collapsible-section">
+  <h2 class="c-collapsible-section-heading" data-accordion-role="tab">Targeted Advertising Disclosures</h2>
+
+  <div data-accordion-role="tabpanel">
+    <p>In the Reporting Period, and during the time we started targeted advertising, we have placed the following targeted advertisements.</p>
+
+    <h3>Mozilla</h3>
+
+    <ul class="mzp-u-list-styled">
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/north-america-mozilla-may-2021-campaign.pdf">
+          Mozilla North America May 2021 Campaign
+        </a>
+      </li>
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/mozilla-european-june-2022-campaign.pdf">
+          Mozilla European June 2022 Campaign
+        </a>
+      </li>
+    </ul>
+
+    <h3>Firefox</h3>
+
+    <ul class="mzp-u-list-styled">
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/firefox-north-america-july-2021-campaign.pdf">
+          North America July 2021 Campaign
+        </a>
+      </li>
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/firefox-europe-october-2021-campaign.pdf">
+          Europe October 2021 Campaign
+        </a>
+      </li>
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/firefox-north-america-october-2021-campaign.pdf">
+          North America October 2021 Campaign
+        </a>
+      </li>
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/firefox-north-america-november-2021-campaign.pdf">
+          North America November 2021 Campaign
+        </a>
+      </li>
+      <li>
+        <a href="https://assets.mozilla.net/pdf/transparency-report/h1-2022/firefox-north-america-february-2021-instagram-campaign.pdf">
+          North America February 2022 Campaign
```

Comment:
Note to self: one of the years here is incorrect, will update when we get confirmation.